Docs: Agregar resumen del capitulo 11

IIKUYY opened the pull request:
## what

- Agregar el resumen del capitulo 11 junto con sus ejemplos

## why

- Continuar con las tareas dadas

## references
- http://diveintohtml5.info/index.html

IIKUYY committed:
Docs: Agregar resumen del capitulo 11

## what

- Agregar el resumen del capitulo 11 junto con sus ejemplos

## why

- Continuar con las tareas dadas

## references
- http://diveintohtml5.info/index.html

semanticmx commented on Ch11/Ejemplo/index.html:
```diff
@@ -0,0 +1,44 @@
+            content="width=device-width, initial-scale=1">
+    </head>
+    <body>
+    <header 
```
Aquí el formato va así:

```html
<header id="title">
    Ejemplo del uso del API de historial
</header>
```

porque header solo tiene un atributo

semanticmx commented on Ch11/Ejemplo/index.html:
```diff
@@ -0,0 +1,44 @@
+                <figcaption>Control de Xbox</figcaption>
+        </figure>
+    </aside>
+    <script 
```
muy bien

semanticmx commented on Ch11/Ch11.md:
```diff
@@ -0,0 +1,113 @@
+## Introducción
```
solo falta agregar los screenshots del navegador y listo.

IIKUYY committed:
chores: Realizar las corrección necesarias

## what

- Agregar screenshot de otro navegador
- Corregir código

## why

- Mostrar variedad de ejemplos
- Corregir la semántica del código

## references

-

IIKUYY commented:
Se realizaron los cambios
